templates/thank-you.html

```diff
@@ -80,6 +80,15 @@ <h2 data-qa="view-submitted-response-title">{{ _("Get a copy of your answers") }
     {% endif %}
   {% endif %}
 
+  {% set form = content.form %}
```

Comment:
Minor call this `confirmation_email_form` as we will have other form on this page soon (feedback).

Reply:
I'm not able to make this change unless we change `error-panel.html` template (add some logic like `if confirmation_email_form`). `error-panel.html`  is universally used by all `forms` so I'm not sure if we want to litter it with extra logic. Workaround would be to create another `error-panel.html` just for emails, but I am not sure if we want that. For now I am changing `partials/email-form.html` to `partials/confirmation-email-form.html` as it has some `Confirmation` specific content and it's the least invasive.

Reply:
I'm not seeing the issue. Why does `error-panel.html` need changing?.
The error template should be given `form` regardless of which parent template calls it.

Given we will have more than one form, I would call this `confirmation_email_form` in the context before it is even passed to the view. 

It is not an issue in `confirm-email.html` because that will only every have one form, so the line can stay unchanged.

Reply:
`confirmation-email.html` -> `error-panel.html. Check `test_confirmation_email.json` and get to send another email panel and send black email.

Reply:
`confirmation-email.html` will need to change for a scenario when you try to send another email from `/submitted/confirmation-email/send`, then you will get the error:
<img width="1199" alt="Screenshot 2023-11-22 at 07 56 09" src="https://github.com/ONSdigital/eq-questionnaire-runner/assets/53475968/9456d884-3053-4e63-a43b-8d51bbdce1fe">

Reply:
Just some more context, if we don't modify error message (if we change the form name) then we will end up with an empty error panel. This is on second confirmation email.
![Uploading Screenshot 2023-11-22 at 09.02.56.png…]()

Reply:
<img width="400" alt="Screenshot 2023-11-22 at 09 02 56" src="https://github.com/ONSdigital/eq-questionnaire-runner/assets/53475968/522e70fb-0f16-488c-bd2b-af6f2bcab3e1">

Reply:
Here are the changes needed to support multiple forms on the thank you page. No changes needed to any other html files. It was easier for me to push up the changes to a copy branch to demo the change. Only 12 lines of change.
https://github.com/ONSdigital/eq-questionnaire-runner/compare/add-confirmation-email-support-non-census-thank-you-page...add-confirmation-email-support-non-census-thank-you-page-multiple-forms?expand=1

Any questions, let me know.

Reply:
That solves the whole problem, pushed up now, thanks!